client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import {Box, Typography, styled, keyframes  } from '@mui/material';
+// import { useInView } from 'react-intersection-observer';
+
+const UkraineTextInfo = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
+    [theme.breakpoints.up('tablet')]: {
+        padding: '10px',
+        fontSize: '18px',
+        lineHeight: '1.2',
+    },
+
+    [theme.breakpoints.up('laptop')]: {
+        fontSize: '18px',
+        padding: '20px'
+    },
+}));
+
+const AboutUkraineContainer = styled((props) => <Box {...props} />)(({ theme }) => ({
+    backgroundColor: '#EDEDED',
+    borderRadius: '20px',
+    zIndex: 1,
+    transition: '2s',
+    position: 'absolute',
+    top: '12%',
+    left: '40%',
+    width: '200px',
+    marginRight: '30px',
+    padding: '20px',
+   
+    '&:hover':{
+        backgroundColor: '#0499dd',
+        animation: `
+        -webkit-transform: translateX(-60px);
```

Comment:
please use camel case not kebab case

```suggestion
        webkitTransform: translateX(-60px);
```

Reply:
done

Reply:
not done.. correct syntax is  webkitTransform not  -webkitTransform. 

please fix

Reply:
Done